packages/itwinui-react/src/core/ColorPicker/ColorBuilder.tsx

```diff
@@ -305,6 +307,7 @@ export const ColorBuilder = React.forwardRef((props, ref) => {
 
       {showAlpha && (
         <Slider
+          aria-label='Opacity slider'
```

Comment:
`aria-label` does nothing on the Slider. it is getting applied on the outer Slider container, which is a generic div.

Reply:
I'm a little unsure by what do you mean here. But if you intend to pass an `aria-label` to the slider, you would need to pass it to the thumb props. This is because the thumbs are the actual input tags that need to have an accessible name.

https://github.com/iTwin/iTwinUI/blob/bd12e4917550aa39d4144d41a72ffcd6c3d53242/examples/Slider.tooltipnone.tsx#L30-L34

(from PR #1474)

Reply:
Implemented with [this ](https://github.com/iTwin/iTwinUI/pull/1470/commits/630a2a2f2693c658dd0cbaeda68cc3fe1b5686a5)commit from yesterday.